packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -128,61 +128,65 @@ const SidebarLink: FC<SidebarLinkProps> = (props) => {
             [styles.open]: open || selected,
           })}
         >
-          {props.subLinks.map((l) => {
-            if (l.superAdmin && !admin) return null;
+          {props.subLinks
+            .filter(
+              (subLink) => !subLink.feature || growthbook.isOn(subLink.feature)
```

Comment:
Should we also 404 the integrations/slack page itself if the feature is off? Personally I might not bother, just calling it out...

Reply:
I think it's ok if we leave the route alone. It's easier for us to test in prod if we wanted to before releasing the feature.